code/modules/codex/codex_mob.dm

```diff
@@ -0,0 +1,18 @@
+/mob/proc/can_use_codex()
+	return TRUE
+/*
```

Comment:
Commented out code :whip:

Reply:
So it's all still there for if we want to tone the codex down a bit.